loops.js

```diff
@@ -10,8 +10,10 @@
 // Tip: je kunt de .repeat() methode gebruiken om een karakter een n aantal keer te herhalen...
 // Dit heb je nog niet geleerd, maar bekijk hiervoor dit MDN-artikel eens: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat
 // ==========================================
-
-
+for(let i=1; i <= 5; i++){
+    let a = "*".repeat(i);
+    console.log(a);
```

Comment:
Het gebruik van console.log om scores te printen is eenvoudig en effectief. De for-lus is begrijpelijk en duidelijk.